app/grandchallenge/algorithms/views.py

```diff
@@ -647,17 +644,30 @@ class AlgorithmImageViewSet(ReadOnlyModelViewSet):
     filterset_fields = ["algorithm"]
 
 
-class JobViewSet(ReadOnlyModelViewSet):
+class JobViewSet(
+    CreateModelMixin, RetrieveModelMixin, ListModelMixin, GenericViewSet
```

Comment:
Do you test this from the views? I would have thought that the users would need the global `add_job` permission otherwise it would be rejected by `DjangoObjectPermissions`?

Reply:
Ah! That would be it. You can add the model level permissions, then no need to change anything in the ViewSet.

Any authenticated user can create a job as you do the checks on algorithm use later. So, like this but for jobs, not answers: https://github.com/comic/grand-challenge.org/blob/master/app/grandchallenge/reader_studies/apps.py#L27-L35, remember to register the function on post migrate: https://github.com/comic/grand-challenge.org/blob/master/app/grandchallenge/reader_studies/apps.py#L46